cpan/flatpak-cpan-generator.pl

```diff
@@ -21,7 +21,7 @@ sub scan_deps {
 
   for (@deps)
   {
-      s/^Successfully installed (.*)/$1/;
+      s/^Successfully installed (.*?)(?:\s+\(.*)?$/$1/;
```

Comment:
If different things can be put after the package name, would it be easier to just do something like `(\S+).*`? That way this would work out on any other formats that cpanminus puts there.

Reply:
Sure, it makes the regexp simpler as well. :)